### PR TITLE
Fix Absolute Encoders

### PR DESCRIPTION
**Pull request by alicekuznetsov**
Needs to be tested and the rollover value for the climber is currently way off

BUT TURRET WORKED IN SIM!!!

### CODE REVIEW

**Comment by alicekuznetsov** [comment]
Test this today if possible. I won't be there, so heres the math

image

File: src/main/java/frc/robot/RobotPreferences.java [hunk 1]
```diff
@@ -25,25 +25,30 @@
...
      * <b>Units:</b> Degrees
      */
     public static final SN_DoublePreference climberMotorReverseLimit = new SN_DoublePreference(
         "climberMotorReverseLimit", 0);
     public static final SN_DoublePreference climberMotorUpSpeed = new SN_DoublePreference(
-        "climberMotorUpSpeed", 0.2);
+        "climberMotorUpSpeed", 0.6);
     public static final SN_DoublePreference climberMotorDownSpeed = new SN_DoublePreference(
-        "climberMotorDownSpeed", -0.2);
+        "climberMotorDownSpeed", -0.6);
```
Comment (TaylerUva) [inline src/main/java/frc/robot/RobotPreferences.java]: Are these used anymore now that we are using set points?

**Review by TaylerUva — DISMISSED** [review]
Approved, just one question comment. We can address it in another PR if need be

File: src/main/java/frc/robot/RobotPreferences.java (hunk 1 above)
Comment (alicekuznetsov) [reply src/main/java/frc/robot/RobotPreferences.java]: Nope

**Comment by alicekuznetsov** [comment]
There are a ton of climber/intake preferences that need to be merged into one subsystem... after phr

**Review by TaylerUva — APPROVED** [review]

**Pull request merged**